src/utils/rates.js

```diff
@@ -100,7 +100,7 @@ export const getAssetValueFromFiat = (
 };
 
 export const fiatInvestmentBalance = (balance: BigNumber, rates: RatesPerChain, currency: Currency) => {
-  const nativeAssetRate = rates[nativeAssetPerChain[CHAIN.ETHEREUM].address];
+  const nativeAssetRate = rates?.[nativeAssetPerChain[CHAIN.ETHEREUM].address];
```

Comment:
What happens below if rates is null and in the line below, we pass `parseFloat(assetBalance) * null / null)` ?

Reply:
Got NaN. Got NaN. I have updated the conditions here. Now gets 0

Reply:
Thank you